pkg/driverbuilder/builder/ubuntu.go

```diff
@@ -353,8 +353,10 @@ func ubuntuGCCVersionFromKernelRelease(kr kernelrelease.KernelRelease) string {
 		}
 		return "6"
 	case 5:
-		if kr.PatchLevel >= 13 {
+		if kr.PatchLevel >= 11 {
 			return "10"
+		} else if kr.PatchLevel >= 19 {
```

Comment:
Gotta reverse the order of these `if` statements here, otherwise the second won't execute.

Reply:
lol, am dumb, 2 sec